myapp/app/controllers/tasks_controller.rb

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 
 class TasksController < ::ApplicationController
+  SORTABLE_FIELDS = %w(id created_at)
+  SORTABLE_ORDERS = %w(asc desc)
```

Comment:
not sure I get what you mean [here](https://github.com/Fablic/training/pull/1470#discussion_r1051886053), is that something like this...?

Reply:
Yeah, something like that! 
Thanks for fixing